build/docker-extension/Makefile

```diff
@@ -0,0 +1,33 @@
+# Makefile for development purpose
+
+.PHONY: local build
+local: clean build install
+remote: clean build-remote install-remote
+
+ORG=portainer
+VERSION=0.0.0
+IMAGE_NAME=$(ORG)/portainer-docker-extension
+TAGGED_IMAGE_NAME=$(IMAGE_NAME):$(VERSION)
+
+clean:
+	-docker extension remove $(IMAGE_NAME)
+	-docker rmi $(IMAGE_NAME):$(VERSION)
+	-docker rmi $(IMAGE_NAME)
+
+build:
+	docker buildx build -f build/linux/Dockerfile --load --build-arg TAG=$(VERSION) --build-arg PORTAINER_IMAGE_NAME=$(IMAGE_NAME) --tag=$(TAGGED_IMAGE_NAME) .
+
+install:
+	docker extension install $(TAGGED_IMAGE_NAME)
+
+build-remote:
+	docker buildx build -f build/linux/Dockerfile --push --builder=buildx-multi-arch --platform=windows/amd64,linux/amd64,linux/arm64 --build-arg TAG=$(VERSION) --build-arg PORTAINER_IMAGE_NAME=$(IMAGE_NAME) --tag=$(TAGGED_IMAGE_NAME) .
+
+install-remote:
+	docker extension install $(TAGGED_IMAGE_NAME)
+
+multiarch:
+	docker buildx create --name=buildx-multi-arch --driver=docker-container --driver-opt=network=host
+
+portainer:
+	yarn build
```

Comment:
it might be a good idea to either have an `.md` file or explain here in comments how to use these tasks, when and where (i.e. `multiarch` and `portainer`)